technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note.csl

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Technische Universität Dresden - Finanzwirtschaft und Finanzdienstleistungen (note)</title>
+    <title-short>TUD Betriebswirtschaftslehre (note)</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note</id>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-finanzwirtschaft-und-finanzdienstleistungen-note" rel="self"/>
+    <author>
+      <name>Jens Mittelbach</name>
+      <email>jens.mittelbach@slub-dresden.de</email>
+      <uri>http://www.slub-dresden.de</uri>
+    </author>
+    <contributor>
+      <name>Julian Onions</name>
+      <email>julian.onions@gmail.com</email>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="generic-base"/>
+    <summary>Zitierstil entsprechend den Zitierrichtlinien des Lehrstuhls Finanzwirtschaft und
+Finanzdienstleistungen der Fakultät Wirtschaftswissenschaften, Technische Universität Dresden. Stil basiert auf dem Zitierstil Harvard Reference format 1 (author-date)</summary>
```

Comment:
Add
```
<link href="http://www.zotero.org/styles/harvard1" rel="template"/>
```
above and you can delete the last sentence here. Moreover, you can then delete the contributor as well, because he is credited in the linked style.